lock.go

```diff
@@ -134,6 +118,29 @@ func (l *Lock) MarshalJSON() ([]byte, error) {
 	return buf.Bytes(), err
 }
 
+// TODO(carolynvs) this should be moved to gps
```

Comment:
Let's make an issue for this. (I'm not quite fully convinced, but still, an issue is the thing to do.)